elements/pfe-tabs/demo/index.html

```diff
@@ -173,8 +173,8 @@ <h2>Content 3</h2>
       </pfe-tabs>
     </section>
     <section class="example">
-      <h2>Style: Vertical earth</h2>
-      <pfe-tabs vertical pfe-variant="earth">
+      <h2>Style: Vertical earth </h2>
+      <pfe-tabs pfe-align="center" >
```

Comment:
Noting this from the demo - I think you mentioned this one needed to have the `vertical` attribute added back.

Reply:
@starryeyez024 bump ^